src/codegen/riscv64/cpu-riscv64.cc

```diff
@@ -16,22 +16,7 @@ namespace internal {
 
 void CpuFeatures::FlushICache(void* start, size_t size) {
 #if !defined(USE_SIMULATOR)
-  // Nothing to do, flushing no instructions.
-  if (size == 0) {
-    return;
-  }
-
-#if defined(ANDROID) && !defined(__LP64__)
-  // Bionic cacheflush can typically run in userland, avoiding kernel call.
-  char* end = reinterpret_cast<char*>(start) + size;
-  cacheflush(reinterpret_cast<intptr_t>(start), reinterpret_cast<intptr_t>(end),
-             0);
-#else   // ANDROID
-  long res;  // NOLINT(runtime/int)
-  // FIXME: RISCV porting
-  res = 0;
-  if (res) FATAL("Failed to flush the instruction cache");
-#endif  // ANDROID
+  __builtin___clear_cache(start, (char *)start + size);
```

Comment:
I think this is related to https://github.com/llvm/llvm-project/commit/99a8cc2b7d89ff73a20aa3e0317fee8b5cb5cecb :)